app/views/applications/personal_statement.html

```diff
@@ -17,8 +17,18 @@ <h1 class="govuk-heading-l">{{ title }}</h1>
 
 
       {% set guidanceHtml %}
-        <p class="govuk-body">You should write between 500 and 1000 words for your personal statement.</p>
-        <p class="govuk-body">Here are some examples you could write about (they are just a guide, you do not need to write about all of them):</p>
+
+        {% if lastApplication %}
+          <div class="govuk-inset-text">
+           <p class="govuk-body">We've copied your personal statement from your previous application.</p>
+
+           <p class="govuk-body">You can make changes to your statement for this application, or leave it the same.</p>
```

Comment:
Do you think we should try and steer them into making it unique? Something like:

You can use the statement from your previous application to get you started, but it's a good idea to try and tailor your statement to suit the course you're applying for.

Reply:
I'm not sure, this is something I think we should ask about in testing, would they even tailor it? Do they think its necessary? At the moment they don't, so is this something we should enforce, is that right/putting more work onto the candidates?